ckanext/restricted/tests/test_plugin.py

```diff
@@ -273,3 +273,39 @@ def test_user_can_see_only_accessible_resource_in_package_search(self):
         assert package['num_resources'] == 1
         assert len(package['resources']) == 1
         assert package['resources'][0]['id'] == other_resource['id']
+
+    def test_org_member_see_resources_package_search(self):
```

Comment:
I'd rename the name of the method to `test_org_member_see_only_accessible_resources_in_package_search`
and then you can discard the comment.